src/WpfMath/TexFormulaParser.cs

```diff
@@ -317,7 +318,7 @@ private static TexFormula ConvertRawText(SourceSpan value, string textStyle)
             return formula;
         }
 
-        internal static SourceSpan ReadElementGroup(SourceSpan value, ref int position, char openChar, char closeChar)
+        internal static SourceSpan ReadArgumentGroup(SourceSpan value, ref int position, char openChar, char closeChar)
```

Comment:
I don't like these renames. I suggest to change them back. These "groups" aren't always the "arguments" (as in command arguments): sometimes they're just some groups/items of stuff. It was a motivation for initial naming, and I still back it up.